src/components/pager/pager.style.js

```diff
@@ -120,7 +116,7 @@ const StyledPagerSizeOptionsInner = styled.div`
 
 const StyledPagerNavigation = styled.div`
   display: flex;
-  flex: 1 1 auto;
+  flex: 1 1 100%;
```

Comment:
Does this need to change to 100%?

Reply:
See comment above, I'll revert it back as the change isn't needed anymore